.gitmodules

```diff
@@ -1,7 +1,6 @@
 [submodule "fix"]
 	path = fix
 	url = gerrit:GSI-fix
-
-[submodule "libsrc"]
-	path = libsrc
-	url = gerrit:GSI-libsrc
+[submodule "cmake"]
+	path = cmake
+	url = git@github.com:mark-a-potts/CMakeModules
```

Comment:
Is git@github.com:mark-a-potts/CMakeModules the official location for the cmake submodule? If there are changes to the cmake submodule from other users, is a Pull Request to this repository required?

Reply:
You are correct. I will put in a PR to put it into the NOAA-EMC repo.